orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/UpdateLaunchConfigTask.groovy

```diff
@@ -21,6 +21,7 @@ import com.netflix.spinnaker.orca.api.pipeline.Task
 import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution
```

Comment:
I'd be tempted to rename this class to something like AbstractSomethingSomethingLaunchTask and then have two concrete subclasses.

Feels like some slight risk that someone could accidentally break launch template by doing something for launch config

Reply:
That's a good point.